grails-app/migrations/install/changelog-create-schema.groovy

```diff
@@ -2342,7 +2346,9 @@ databaseChangeLog = {
                 constraints(nullable: "false")
             }
 
-            column(name: "request_item_id", type: "CHAR(38)")
+            column(defaultValue: "", name: "request_item_id", type: "CHAR(38)") {
+                constraints(nullable: "false")
```

Comment:
It feels wrong to use defaultValue = "" and nullable = false. Do we need to use defaultValue? I assume this is part of the dbm-gorm-diff logic, but we should figure out if 1) we need it 2) if not, how to disable it.

Reply:
I think that it should be the former version (without defaultValue), but will investigate it a bit more.